### DISCUSSION TITLE
Complementary Notes (In case you are having issues)

### DISCUSSION THREAD

**Discussion by 51av0sh** [willswire/unifi-ddns#57] [Q&A]
I ran into a few issues and took me about a week to troubleshoot and fix them all. Since documentation is limited I wanted to offer my experience in case someone has similar problems.

1. Starting with installation. I was getting an error when running the `Deploy Workers` script. I found out that the GHA was not running from the forked repo. To force the run all I did was edit he `deploy.yml` file (added a space and then removed it).  That triggered the workflow and it installed the worker in CF.

2. I have a UDM SE and had to remove the `https://` from the server field as it was not working

3. I have a wildcard DNS entry and a root domain entry. To update both at he same time you can add a list of comma separated domains to the Host field like `domain.com,*.domain.com`. Make sure you have no spaces

4. Some said that `dyndns` is missing from the UDM but it's there for me in my UDM SE running network application 8.0.28. According to some, if you are missing `dyndns` you can use `dyn`. Have not confirmed this.

5. If you want to test this script you have a few options
     1. Ask your ISP for a new IP.
     2. Change the modem mac. You might need to contact your ISP to register the modem again.
     3. Use `inadyn` (easiest but needs SSH). See [here](https://github.com/workerforce/unifi-ddns/issues/42#issuecomment-1790029386)

6. Proxied DNS works fine with his

**Comment by lobrau** [comment]
Thank you, the wildcard domain helped me immensely.

**Comment by partytimeexcellent** [comment]
I'm stuck at the very first step. Docs say to "Deploy the Worker", but when I click the link, I'm asked to log in with my user name and API Token. Unfortunately there's no explanation of where this API Token is supposed to be from. I tried creating the DNS Zone Editor token described in the next step, but it's not accepted. So how did you handle this? Do I need to create a different API Token just to get started?

**Reply by willswire** [reply]
Please check out the latest release! I've taken the time to update the documentation and correct several issues reported by the community.

**Comment by willswire** [comment]
Thank you for sharing these helpful tips and tricks! I've attempted to include most of them in the new FAQ in `/docs/faq.md`. If you have any more, please consider contributing them to the community!
